### DISCUSSION TITLE
오픈채팅방 왕관 여부 를 체킹할수있나요 ?

### DISCUSSION THREAD

**Discussion by difmwng3** [storycraft/node-kakao#400] [Q&A]
메세지를 받아오면서 이사람이 왕관(부방장,방장) 인 사람인지
체크를 하고싶은데 가능한가요 ?

**Comment by jipsoft2** [comment]
v3인가요 v4인가요?

**Reply by difmwng3** [reply]
v4입니다.

**Reply by jipsoft2** [reply]
v4는 아직 안 써봐서 잘 모르겠네요 ㅠㅠ

**Comment by generalxd** [comment]
Type 관련 Struct Property를 체크해보시기 바랍니다.

**Comment by storycraft** [comment]
TalkOpenChannel 객체에서 유저 정보를 가져오면 perm 속성을 통해 권한을 확인 할 수 있습니다. https://github.com/storycraft/node-kakao/blob/488d5a129a95ae810bbd56935db2f517ba09e7ac/src/openlink/open-link-user-info.ts#L57

**Reply by storycraft** [reply]
오픈 채팅방 이벤트만 받으려면 #320 을 참고하세요.

**Comment by mschadev** [comment]
```js
function kakaotalkMessageCallback(data,channel){
const sender = data.getSenderInfo(channel);
  if (!sender) { 
    return;
  }
 if(sender.perm === OpenChannelUserPerm.MANAGER ||
    sender.perm === OpenChannelUserPerm.OWNER){ 
//
  }
}
```
